web/templates/right_sidebar.hbs

```diff
@@ -2,10 +2,12 @@
     <div class="right-sidebar-items">
         <div id="user-list">
             <div id="userlist-header">
-                <h4 class='right-sidebar-title' id='userlist-title'>
-                    {{t 'USERS' }}
-                </h4>
-                <i id="user_filter_icon" class="fa fa-search"></i>
+                <span id="userlist-header-search">
+                    <h4 class='right-sidebar-title' id='userlist-title'>
+                        {{t 'USERS' }}
+                    </h4>
+                    <i id="user_filter_icon" class="fa fa-search"></i>
```

Comment:
Should we change this to `.zulip-icon-search` here?

Reply:
There's a different design for search, so I think it makes sense to just not touch it for now.